test/performance/benchmarks/deployment-probe/continuous/main.go

```diff
@@ -42,7 +44,7 @@ import (
 	networkingclient "knative.dev/networking/pkg/client/injection/client"
 	"knative.dev/pkg/test/mako"
 	asv1alpha1 "knative.dev/serving/pkg/apis/autoscaling/v1alpha1"
-	"knative.dev/serving/pkg/apis/serving/v1beta1"
+	"knative.dev/serving/pkg/apis/serving/v1"
```

Comment:
<!--gotool-bot-->
Format Go code:
```suggestion

```

Reply:
I don't think that's right.

Reply:
Can you force push with a `v1` alias?  Possibly a `goimports` bug...

Reply:
Repushed